pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
```

Comment:
No EOL here.  I notice that vscode is bad about that, is that the culprit?

Reply:
It turns out that there is an option to automatically insert a newline, but it's off by default.  I've enabled it, so hopefully better now.